cli.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
+/* eslint no-console: "off" */
+
+const meow = require('meow');
+const chalk = require('chalk');
+const loadJsonFile = require('load-json-file');
+const path = require('path');
+const Octokit = require('@octokit/rest');
+const LabelSyncer = require('./src/label-syncer');
+const repositoryParser = require('./src/model/repository');
+const labels = require('./src/model/labels');
+
+const github = new Octokit({
+  timeout: 0,
+  headers: {
+    'user-agent': '@rdohms/gh-labeller',
+  },
+});
+
+const cli = meow(
+  `
+Usage
+    $ sync-labels -t <token> -c <config> <repository>
+
+    Options
+      -d           Dry run, don't actually copy anything
+      -t, --token  Token to authenticate with GitHub API
+      -c, --config Configuration file that holds desired labels
+
+    Examples
+      $ sync-labels -t token -c labels.json rdohms/label-manager
+`,
+  {
+    flags: {
+      dryrun: {
+        type: 'boolean',
+        alias: 'd',
+        default: false,
+      },
+      token: {
+        type: 'string',
+        alias: 't',
+      },
+      config: {
+        type: 'string',
+        alias: 'c',
+      },
+    },
+  }
+);
+
+if (!cli.flags.token) {
+  cli.showHelp(1);
+}
+
+if (cli.input.length < 1) {
+  cli.showHelp(1);
+}
+
+const [repository] = cli.input;
+
+const options = {
+  dryRun: cli.flags.dryrun,
+  token: cli.flags.token,
+  config: cli.flags.config,
+};
+
+if (cli.flags.dryrun) {
+  console.log(chalk.yellow('Dry run, no labels are copied for real:'));
+}
+
+loadJsonFile(path.normalize(options.config))
+  .then(json => {
+    const [owner, repo] = repositoryParser.parse(repository);
```

Comment:
Should we run this directly after line 60, so we break soon if the provided repository argument is not correct?